anxcloud/data_source_vsphere_locations.go

```diff
@@ -35,10 +36,6 @@ func dataSourceVSphereLocationsRead(ctx context.Context, d *schema.ResourceData,
 		return diag.FromErr(err)
 	}
 
-	if id := uuid.New().String(); id != "" {
-		d.SetId(id)
-		return nil
-	}
-
-	return diag.Errorf("unable to create uuid for locations data source")
+	d.SetId(strconv.FormatInt(time.Now().Round(time.Hour).Unix(), 10))
```

Comment:
Could you add location code and organization to the ID, the same as with tag.

Reply:
done